pkg/apb/rhcc_registry.go

```diff
@@ -127,7 +126,17 @@ func (r RHCCRegistry) imageToSpec(image *Image) (*Spec, error) {
 		return nil, err
 	}
 
+	if conf.Config == nil {
+		r.log.Error("Did not find v1 Manifest in image history. Skipping.")
+		return nil, errors.New("Error: v1 Manifest does not exist for this image")
```

Comment:
The error you're returning isn't being used by the caller. Since the apb is getting skipped, I think it should be ```return nil, nil```